### DISCUSSION TITLE
Aide pour récupérer mon GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by fedigz** [community/community#65493] [GitHub Education]
### Hi everyone,

Je me permets de vous contacter car j'ai rencontré des difficultés lors de la tentative de récupération de mon GitHub Student Developer Pack. J'apprécie vraiment l'opportunité offerte par GitHub Global Campus et je suis enthousiaste à l'idée d'utiliser les ressources et les avantages du pack étudiant.

Cependant, malgré mes efforts, je n'ai pas réussi à accéder aux avantages du pack. Je suis actuellement inscrit sur GitHub Global Campus en tant qu'étudiant, mais je ne sais pas quelles sont les étapes exactes à suivre pour récupérer les offres du pack. J'aimerais donc solliciter votre aide pour comprendre comment procéder.

Pourriez-vous s'il vous plaît m'indiquer les étapes précises que je dois suivre pour récupérer mon GitHub Student Developer Pack ? Y a-t-il des conditions particulières que je dois remplir ou des informations spécifiques que je dois fournir pour valider ma demande ?

Je vous remercie infiniment de l'attention que vous porterez à ma demande. Je suis impatient de pouvoir accéder aux ressources et avantages offerts par GitHub Global Campus.

**Comment by geoffreylgv** [comment]
Bonjour @fedigz, vous êtes à la bonne porte
## Conditions et comment avoir son pack etudiant de GitHub
1. Être inscrit.e dans une université ou institution
2. Avoir un compte GitHub, et ce compte doit dater de plus de 6 mois
3. Avoir un mail étudiant (edu ou un mail contenant le nom de domain de votre ecole/université/instition)
Exemple : Mon université, c'est **Université de Lomé**, située au **Togo**, avec domaine **_www.univ-lome.tg_**
Dans ce cas, l'étudiant aura un mail comme suit : `prenom.nom@univ-lome.tg`

>[!IMPORTANT]
La génération mail peut être aussi différente selon l'université ou administrateur du service informatique de l'école.
Dans le cas où vous n'avez pas de mail edu, vous devez avoir **votre carte d'étudiant et à jour**, ceci vous permettra de **pouvoir vérifier votre status étudiant**
4. Sur le formulaire de [GitHub Education Developer Pack](https://education.github.com/discount_requests/application), vous allez vous connecter avec votre compte GitHub
5. Après avoir connecté son compte GitHub à GH Education, sélectionner votre mail si vous n'avez pas de mail  edu,  au cas contraire ajouter votre mail edu
6. Rechercher votre institution/École/université et sélectionnez cette dernière
7. Écrire une petite note de motivation pour montrer comment vous comptez utiliser GitHub
8. Soumettez et passez à la vérification de votre status étudiant
9. Une fois votre compte vérifié et accepté, vous recevrez un mail dans les minutes qui suivent ou dans les jours, sans confirmation, tu peux contacter le support de GitHub, ou pose la question ici, plus [d'info ici](https://github.com/orgs/community/discussions/22701)
10. Et voilà, vous avez accès à votre pack étudiant sur GH après vérification, 
>[!NOTE]
il suffit de vous rendre sur [Education GitHub](https://education.github.com)
Se connecter avec son compte GH et puis c'est bon, [vous avez vos outils](https://education.github.com/pack/offers)

J'espère que ceci vous aide
